### PR TITLE
[amp-story-player] Fix import error

### LINKED ISSUE #28355: Uncaught ReferenceError: module$build_system$global_configs$caches_json is not defined
`amp-story-player` is throwing the following error:

```
Uncaught ReferenceError: module$build_system$global_configs$caches_json is not defined

```

Causing the player to stop loading.

This is because in #27658 we import the `caches.json` found in `build-system/`, but it looks like it's not bundled in production.

/cc @ampproject/wg-stories

### PR DESCRIPTION
**Pull request by Enriqe**
Fixes #28355

### CODE REVIEW

File: src/amp-story-player/amp-story-player-impl.js [hunk 1]
```diff
@@ -14,7 +14,6 @@
  * limitations under the License.
  */
 
-import * as ampCaches from '../../build-system/global-configs/caches.json';
 import * as ampToolboxCacheUrl from '@ampproject/toolbox-cache-url';
```
Comment (jridgewell) [inline src/amp-story-player/amp-story-player-impl.js:17]: This is still undefined when building. You'll need to add it to the `sources.json`

Comment (Enriqe) [reply src/amp-story-player/amp-story-player-impl.js:17]: Done.

**Comment by Enriqe** [comment]
Adding fix by @jridgewell to the regex checking for unknown dependencies

**Comment by gmajoulet** [comment]
Given that this will go straight to production, should we split it in two with the code to cherrypick, and another PR with the fix for unknown dependency?

**Comment by Enriqe** [comment]
@jridgewell asked me to include this change in this PR

**Comment by Enriqe** [comment]
Split it and sent the fix for the dep checks in #28366

**Pull request merged**